package.json

```diff
@@ -25,7 +25,7 @@
     "csrf": "^3.0.0",
     "cytube-common": "git://github.com/CyTube/cytube-common",
     "cytube-mediaquery": "git://github.com/CyTube/mediaquery",
-    "cytubefilters": "git://github.com/calzoneman/cytubefilters#67c7c69a",
+    "cytubefilters": "git://github.com/davidsiaw/cytubefilters",
```

Comment:
This probably isn't going to fly. If you need to change something there to make this work, make a PR there.